.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: 'Create a report to help us improve'
+title: '[BUG]: '
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: '### Before you open an issue, please verify if a similar one already exists or has been closed before. More details about the process of contributing can be found in [CONTRIBUTING.md](https://github.com/TheAlgorithms/JavaScript/blob/master/CONTRIBUTING.md).'
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Explain what the problem is.
+    validations:
+      required: true
+  - type: textarea
+    id: expectedbhv
+    attributes:
+      label: Expected Behavior
+      description: Describe what was the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: actualbhv
+    attributes:
+      label: Actual Behavior
+      description: Describe what actually happens.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce (if applicable)
+      description: List steps to reproduce the behavior.
+      placeholder: |
+        1. 
+        2. 
+        3. 
+        4.
+    validations:
+      required: false
+  - type: textarea
+    id: specifications
+    attributes:
+      label: Specifications
+      description: Specify version, platform, and web browser (if applicable).
```

Comment:
This doesn't really make sense. This repo is not for a browser; it's for Node.js. The Node.js version is *very unlikely* to be the cause of an issue.

Reply:
@appgurueu Which specification details would be useful then? Or should I remove this section entirely?

Reply:
@appgurueu Regarding my previous comment, can you explain what the `Specifications` section should look like?

Reply:
I'd remove it entirely.